File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Hình ngôi sao1/NGOISAO1.cs
using System;

namespace TamGiacSao
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine()); // Nhập số dòng cần in
            for (int i = 1; i <= n; i++) { // Duyệt từ 1 đến n
                for (int j = 1; j <= i; j++) // In sao từ 1 đến i
                    Console.Write("*");
                Console.WriteLine(); // Xuống dòng sau mỗi hàng
            }
        }
    }
}

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Hình ngôi sao1/NGOISAO1.py
n = int(input()) # Nhập số dòng cần in
for i in range(1, n + 1): # Duyệt từ 1 đến n
    for j in range(1, i + 1): # In sao từ 1 đến i
        print("*", end="") # In sao không xuống dòng
    print() # Xuống dòng sau mỗi hàng
